### PR TITLE
adding new shopping cart page

### PR DESCRIPTION
**Pull request by jorgegpa**
This feature adds the new Shopping Cart Page for use in automation

### CODE REVIEW

File: src/main/java/pages/ShoppingCartPage.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ShoppingCartPage {
+    @FindBy(css="a.btn_action")
+    private WebElement checkoutBtn;
+
```
Comment (Ajtrujillom) [inline src/main/java/pages/ShoppingCartPage.java:9]: Please also map the other button

**Review by Ajtrujillom — CHANGES_REQUESTED** [review]

File: src/main/java/pages/ShoppingCartPage.java (hunk 1 above)
Comment (jorgegpa) [reply src/main/java/pages/ShoppingCartPage.java:9]: On it , will map it

Comment (Ajtrujillom) [reply src/main/java/pages/ShoppingCartPage.java:9]: ok

**Review by Ajtrujillom — APPROVED** [review]
changes approved

**Comment by Ajtrujillom** [comment]
this branch had been merged please delete it

**Pull request merged**